.github/workflows/polyval.yml

```diff
@@ -39,7 +39,27 @@ jobs:
           target: ${{ matrix.target }}
           override: true
       - run: cargo build --no-default-features --release --target ${{ matrix.target }}
-  test:
+
+  test_u32:
```

Comment:
Note: split these up into different jobs for each backend. I'm going to "break" each backend individually and make sure each of the backends is actually getting tested.

Reply:
Glad I did that! It seems `test_u32` wasn't actually selecting the right target! 😅 

Fortunately the wasm32 tests still caught breakages in the 32-bit backend.

Reply:
Confirmed `polyval`/`test_u32` now fails if the 32-bit backend is broken: https://github.com/RustCrypto/universal-hashes/pull/75/checks?check_run_id=1177963767

Reply:
Oh joy, now it won't build because of `criterion` 😅 

https://github.com/RustCrypto/universal-hashes/pull/75/checks?check_run_id=1177981676

I'll go ahead and rip `criteron` out (and potentially replace it with a `benches/` directory)

Reply:
Okay, 32-bit backend now builds successfully! 🎉 

I also tested that the CLMUL tests fail if it's broken:

https://github.com/RustCrypto/universal-hashes/pull/75/checks?check_run_id=1178090082

So I'd say we're good to go here.